probe-rs/src/architecture/riscv/communication_interface.rs

```diff
@@ -28,7 +27,7 @@ pub enum RiscvError {
     #[error("Error during read/write to the DMI register: {0:?}")]
     DmiTransfer(DmiOperationStatus),
     /// An error with operating the debug probe occurred.
-    #[error("Debug Probe Error")]
+    #[error("Debug Probe Error: {0}")]
```

Comment:
This is not how it is intended :) The intention is to print the entire error stack. This will print the inner error twice.

Reply:
While this would be nice, I think I started going through these because I experienced losing a lot of inner errors. I'm guessing something might not be completely as intended, or maybe I just didn't notice something :)